src/local/LocalTransport.ts

```diff
@@ -0,0 +1,93 @@
+import assert from 'assert';
+import _, { values, valuesIn } from 'lodash';
+import debug from 'debug';
+import Matcher from './matcher';
+import AutoBus from '../AutoBus';
+
+const log = debug('autobus:transport:local');
+
+export interface Listener<T> {
+  (message: T): void;
+}
+
+export default class LocalTransport {
+  _bus: AutoBus;
+  _wildcardMatchers: Matcher;
+  _channelListeners: Record<string, Listener<any>[]>;
```

Comment:
We don't need this JS convention when it is in TS. Replace the `_` naming convention with making them `private readonly`.